KsApi/models/graphql/adapters/CommentsEnvelope+GraphCommentsEnvelopeTests.swift

```diff
@@ -12,6 +13,22 @@ final class CommentsEnvelope_GraphCommentsEnvelopeTests: XCTestCase {
     XCTAssertEqual(envelope.hasNextPage, true)
     XCTAssertEqual(envelope.cursor, "WzMwNDkwNDY0XQ==")
     XCTAssertEqual(envelope.totalCount, 100)
+    XCTAssertNil(envelope.updateID)
+    XCTAssertEqual(envelope.slug, "jadelabo-j1-beautiful-powerful-and-smart-idex-3d-printer")
+  }
+
+  func testProjectUpdateComments() {
+    let envelope = CommentsEnvelope.commentsEnvelope(from: .template)
+      |> \.updateID .~ 1_234
+
+    XCTAssertEqual(
+      envelope.comments,
+      [Comment.comment(from: .template), Comment.comment(from: .template), Comment.comment(from: .template)]
+    )
+    XCTAssertEqual(envelope.hasNextPage, true)
+    XCTAssertEqual(envelope.cursor, "WzMwNDkwNDY0XQ==")
+    XCTAssertEqual(envelope.totalCount, 100)
+    XCTAssertEqual(envelope.updateID, 1_234)
     XCTAssertEqual(envelope.slug, "jadelabo-j1-beautiful-powerful-and-smart-idex-3d-printer")
```

Comment:
I guess real data would not return both a `slug` and an `updateId` for project update comments, may want the test to reflect this like in `testProjectComments`

Reply:
Also may want to create a new template just for project update comments in `GraphCommentsEnvelope` and use that instead of modifying the existing template meant for project comments.